services/newsfeed/methods/controllers/newsfeed.js

```diff
@@ -13,27 +13,35 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+const {
+	Exceptions: { ServiceUnavailableException },
+} = require('lisk-service-framework');
+
 const config = require('../../config.js');
+const { getNewsfeedArticles } = require('../../shared/newsfeed.js');
 
 const enabledSources = Object.values(config.sources)
 	.filter(({ enabled }) => enabled)
 	.map(({ name }) => name);
 
 const getNewsfeed = async ({ limit, offset, source = enabledSources }) => {
```

Comment:
I think it's better to pass just params here to make it consistent and update the accordingly.
```suggestion
const getNewsfeed = async params => {
```